lib/scrabble/player-class.rb

```diff
@@ -0,0 +1,48 @@
+module Scrabble
+  class Player
+    attr_accessor :name, :plays
```

Comment:
Is there ever a case where `name` or `plays` needs to be modified from the outside? if not, `attr_reader` is technically more correct.